### ISSUE TITLE
Create Useful Alerts in Logentries

### ISSUE THREAD

**Issue by cadecairos** [MozillaFoundation/mofo-devops#341]
### Details
We should leverage alerts on logentries to create an effective defense against undetected issues.

### Info & links
https://logentries.com/

@alicoding and @gideonthomas will be leading this effort, so they can get familiar with the tools, and I will advise and support them. I believe you both have been invited to the account.

**Comment by cadecairos** [comment]
To add to this, we need to find a useful threshold for these alerts. Over the weeked we got lots of notifications by email and in the devops channel. While we probably should fix the large amounts of 503s our apps are causing, until we do so we don't want a flood of alerts. I already took some steps to fix this, but more can be taken.

I also discovered that when you have a single alert that monitors multiple streams, the triggers for it apply globally to all the streams rather than individually, so I'm creating alerts for the more critical Heroku apps we're monitoring (donate, login stuff, publish, thimble, goggles, chat) to make sure we get alerts from them.
